wp-content/themes/access/src/js/modules/screener.js

```diff
@@ -454,6 +454,10 @@ class Screener {
       this._renderRecap();
     }
 
+    let stepTitle = $(section).find('[data-js="step-title"]').text();
+    let baseTitle = 'Eligibility Screener – ACCESS NYC - ';
+    $('title').text(baseTitle + stepTitle);
```

Comment:
Then this line should look like this;
```
$('title').text(`${stepTitle} - ${baseTitle}`);
```
(you might notice I switched from concatenating a string to JavaScript template literals which I think are cool to learn https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)